.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -135,3 +146,21 @@ jobs:
           git push -u origin shared-transformer-$TAG_NAME
 
           hub pull-request -m "chore: upgrade shared transformers to $TAG_NAME"
+
+      - name: Update Helm Chart and Raise Pull Request For Hosted Transformer
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+        run: |
+          cd rudder-devops
+          git checkout -b hosted-transformer-$TAG_NAME ${{steps.extract_branch_name.outputs.branch_name}}
+
+          cd customer-objects/multi-tenant-us
+          yq eval -i ".spec.transformer.image.version=\"$TAG_NAME\"" hostedmtedmt.yaml
```

Comment:
Specify entire path of `hostedmtedmt.yaml` , there maybe multiple files with same name in different directories

Reply:
`customer-objects/multi-tenant-us` This is path of hostedmtedmt.yaml

Reply:
<img width="309" alt="Screenshot 2023-06-20 at 10 00 39 AM" src="https://github.com/rudderlabs/rudder-transformer/assets/60897972/aae6e8a6-2a2c-4945-9760-a4fdda9fb613">

Reply:
overlooked L157 where you are already traversing to the required path